src/NetworkFunction/custom/Models/AzureTrafficCollector.PowerShell.cs

```diff
@@ -0,0 +1,216 @@
+// Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.8.0, generator: @autorest/powershell@3.0.481)
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.NetworkFunction.Models
+{
+    using Microsoft.Azure.PowerShell.Cmdlets.NetworkFunction.Runtime.PowerShell;
+
+    /// <summary>Azure Traffic Collector resource.</summary>
+    [System.ComponentModel.TypeConverter(typeof(AzureTrafficCollectorTypeConverter))]
+    public partial class NetworkFunction
```

Comment:
The class name should be `AzureTrafficCollector`. And it's better to use the models generated under `generated/Models`.